spec/class.dd

```diff
@@ -1241,33 +1241,69 @@ int bar()
         )
 
         $(P The property $(D .outer) used in a nested class gives the
-        $(D this) pointer to its enclosing class. If the enclosing
-        context is not a class, the $(D .outer) will give the pointer
+        $(D this) pointer to its enclosing class. If there is no  enclosing
```

Comment:
Fixed to: "If there is no enclosing class context, .outer would return a pointer to enclosing function frame with void*."